i18n/checkLocaleKeys.js

```diff
@@ -21,6 +21,25 @@ const checkLocaleKeys = () => {
 
     const referenceKeys = extractAllKeys(enFileContent)
 
+    //check if in CI/CD
+    if (process.env.NODE_ENV === 'production') {
+        const missingKeysSummary = findAndReportMissingKeysInNonEnFiles(
+            localeFilesWithoutEnFileName, enFileContent, referenceKeys
+        )
+
+        if (missingKeysSummary.length) {
+            console.error('\x1b[31mMissing keys found:\x1b[0m')
+            missingKeysSummary.forEach(({ file, keys }) => {
+                console.error(`\x1b[33m${file}:\x1b[0m`)
+                keys.forEach(key => console.error(`- ${key}`))
+            })
+            process.exitCode = 1 // Exit with error code
+        } else {
+            console.log('✅ \x1b[32mAll locale files are up to date.\x1b[0m')
```

Comment:
we probably should exit here without an error so it doesn't keep  running the rest of the code for no reason